shared-components/tl-tournament-card.js

```diff
@@ -1,228 +1,291 @@
-import '@material/mwc-button';
-
-import { LoggedUser } from './tl-logged-user.js';
-
-const tournaments = {
-    '(trn)2022-04-15-retail-636546': {
-        id: '(trn)2022-04-15-retail-636546',
-        title: 'Retail market',
-        description: 'Identify the trends among the retail companies and submit your predictions',
-        timeZone: 'CET',
-        registrationStart: '2022-04-09T07:00Z',
-        registrationEnd: '2022-04-12T07:00Z',
-        activityStart: '2022-04-09T07:00Z',
-        activityEnd: '2022-04-15T18:00Z',
-        prizeDistribution: {
-            type: 'fixed',
-            currencyId: 'mglnd-gold',
-            totalValue: 10,
-            shareBetweenTies: true,
-            distribution: [
-                2000, 1000, 500, 300, 200, 100, 100, 100, 100, 100, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 50, 50, 50, 50,
-                50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25,
-                25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25,
-                25, 25, 25, 25, 25, 25, 25, 25, 25, 25
-            ],
-            spreadRest: false
-        },
-    },
-    '(trn)2022-04-15-techno-955159': {
-        id: '(trn)2022-04-15-techno-955159',
-        title: 'Technology market',
-        description: 'Identify the trends among the technology companies and submit your predictions',
-        timeZone: 'CET',
-        registrationStart: '2022-04-09T07:00Z',
-        registrationEnd: '2022-04-12T07:00Z',
-        activityStart: '2022-04-09T07:00Z',
-        activityEnd: '2022-04-15T18:00Z',
-        ruleId: 'mglnd-tournament-rule-2022-04-14',
-        sponsorIds: [],
-        assets: {
-            heroImgCard: 'https://cdn.mogaland.io/assets/data-related/tournament/2022-04-15-techno-955159-400x200.png'
-        },
-        registrationFee: {
-            value: 0,
-            currencyId: 'mglnd-coin'
-        },
-        prizeDistribution: {
-            type: 'fixed',
-            currencyId: 'mglnd-gold',
-            totalValue: 10,
-            shareBetweenTies: true,
-            distribution: [
-                2000, 1000, 500, 300, 200, 100, 100, 100, 100, 100, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 75, 50, 50, 50, 50,
-                50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 50, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25,
-                25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25, 25,
-                25, 25, 25, 25, 25, 25, 25, 25, 25, 25
-            ],
-            spreadRest: false
-        },
-    },
-}
+import { LitElement, html, css } from "lit";
+import "@material/mwc-button";
+
+import { LoggedUser } from "./tl-logged-user.js";
 
 const State = {
-    EMPTY: 'emtpy',
```

Comment:
The change in indentation w/ 2 spaces instead of 4 makes the review much more difficult :( I've to jump up & down to try to figure out what is one of your addition, what is just a reformatting side effect...

Please, indent the code w/ 4 characters and push your update...

Just b/c of the mix of your CSS and the old JS code, I can barely see what's new, what's old...